vonk/releasenotes/releasenotes.rst

```diff
@@ -10,6 +10,145 @@ Release notes Vonk
    releasenotes_old
    security_notes
 
+.. _vonk_releasenotes_300-beta1:
+
+Release 3.0.0-beta1
+--------------------
+
+Vonk 3.0.0 is a major upgrade that incorporates handling FHIR R4. This runs in the same server core as FHIR STU3. See :ref:`feature_multiversion` for background info.
+
+.. attention::
+
+   If you have overridden the PipelineOptions in your own settings, you should review the new additions to it in the appsettings.default.json.
+   In particular we added ``Vonk.Fhir.R4`` that is needed to support FHIR R4.
+
+.. attention::
+
+   MacOS: you may need to clean your temp folder from previous specification.zip expansions. Find the location of the temp folder by running ``echo $TMPDIR``.
+
+Database
+^^^^^^^^
+
+#. All databases: got a new field 'InformationModel'
+#. SQL Server, SQLite: 
+
+   #. vonk.entry got a new column 'InformationModel'.
```

Comment:
, which is set to FHIR3.0 for existing resources

Reply:
Added